src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    addOne: () => {
+      obj.result++;
+    },
+
+    reset: () => {
+      obj.result = 0;
+
+      return obj;
+    },
+
+    add(number) {
+      obj.result += number;
+
+      return obj;
+    },
+
+    subtract(number) {
+      obj.result -= number;
+
+      return obj;
+    },
+
+    multiply(number) {
+      obj.result *= number;
+
+      return obj;
+    },
+
+    divide(number) {
+      obj.result /= number;
+
+      return obj;
+    },
+
+    operate: (operation, number) => {
+      operation(number);
+
+      return obj;
```

Comment:
The `operate` method should accept a callback function and a number, then apply the operation to the `result` property. This implementation is correct, but ensure that the provided callback functions (`add`, `subtract`, `multiply`, `divide`) are correctly implemented to work with this method.